avocado/core/safeloader/__init__.py

```diff
@@ -19,271 +19,16 @@
 
 import ast
 import collections
```

Comment:
This split is more than welcome. And I understand the reasons to keep the code inside `__init__.py`, however I would like to suggest to move this to a different module (i.e: `safeloader/core.py`) and keep here only initialization code, and/or imports.